examples/local/mtp_to_ovito.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+MTP_PREDICTION=./experiments/mtp_find_region/predictions.csv
+LAMMPS_OUTPUT=./experiments/mtp_find_region/dump.si-10000-4.yaml
```

Comment:
This filename is identical to a file now located in `data/mtp_template/mtp_predictions.csv`. Why not move that file here?

Reply:
moved